modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase

```diff
@@ -90,6 +90,90 @@ definition {
 		}
 	}
 
+	@description = "LPS-164582 - Verify CRUD functionality with audit when using object definition entry that has the entry history activated"
+	@priority = 5
+	test AssertCRUDWithObjectEntry {
+		property portal.acceptance = "true";
+
```

Comment:
Same thing here too, let's not add this to acceptance until we're sure it's stable